10/main.js

```diff
@@ -0,0 +1,239 @@
+      auto.yearOfManufactureOfTheCar <= 2009
+    ) {
+      spending += 1.7;
+    }
+    if (
+      2010 <= auto.yearOfManufactureOfTheCar &&
+      auto.yearOfManufactureOfTheCar <= 2018
+    ) {
+      spending += 1.4;
+    }
+    if (
+      2019 <= auto.yearOfManufactureOfTheCar &&
+      auto.yearOfManufactureOfTheCar <= 2022
+    ) {
+      spending += 1.2;
+    }
+    if (auto.engine_type === "Gasoline") {
+      spending += 1.8;
+    } else {
+      spending += 2.2;
+    }
+    if (800 <= auto.mass && auto.mass <= 1100) {
+      spending += 1.5;
+    }
+    if (1101 <= auto.mass && auto.mass <= 1400) {
+      spending += 1.7;
+    }
+    if (auto.mass > 1400) {
+      spending += 2;
+    }
+  }
+  if (auto.type == "Sedan") {
+    if (auto.yearOfManufactureOfTheCar < 1990) {
+      console.log("Авто до 1990 не ремонтуємо.");
+    }
+    if (
+      1990 <= auto.yearOfManufactureOfTheCar &&
+      auto.yearOfManufactureOfTheCar < 2008
+    ) {
+      spending += 2.6;
+    }
+    if (
+      2008 <= auto.yearOfManufactureOfTheCar &&
+      auto.yearOfManufactureOfTheCar <= 2009
+    ) {
+      spending += 2;
+    }
+    if (
+      2010 <= auto.yearOfManufactureOfTheCar &&
+      auto.yearOfManufactureOfTheCar <= 2018
+    ) {
+      spending += 1.8;
+    }
+    if (
+      2019 <= auto.yearOfManufactureOfTheCar &&
+      auto.yearOfManufactureOfTheCar <= 2022
+    ) {
+      spending += 1.5;
+    }
+    if (auto.engine_type === "Gasoline") {
+      spending += 2;
+    } else {
+      spending += 2.5;
+    }
+    if (800 <= auto.mass && auto.mass <= 1100) {
+      spending += 1.6;
+    }
+    if (1101 <= auto.mass && auto.mass <= 1400) {
+      spending += 1.8;
+    }
+    if (auto.mass > 1400) {
+      spending += 2.1;
+    }
+  }
+    if (auto.type == "Station_wagon") {
+      if (auto.yearOfManufactureOfTheCar < 1990) {
+        console.log("Авто до 1990 не ремонтуємо.");
+      }
+      console.log(auto.yearOfManufactureOfTheCar);
+      if (
+        1990 <= auto.yearOfManufactureOfTheCar &&
+        auto.yearOfManufactureOfTheCar < 2008
+      ) {
+        spending += 3;
+      }
+      if (
+        2008 <= auto.yearOfManufactureOfTheCar &&
+        auto.yearOfManufactureOfTheCar <= 2009
+      ) {
+        spending += 2.5;
+      }
+      if (
+        2010 <= auto.yearOfManufactureOfTheCar &&
+        auto.yearOfManufactureOfTheCar <= 2018
+      ) {
+        spending += 2.2;
+      }
+      if (
+        2019 <= auto.yearOfManufactureOfTheCar &&
+        auto.yearOfManufactureOfTheCar <= 2022
+      ) {
+        spending += 2;
+      }
+      if (auto.engine_type === "Gasoline") {
+        spending += 2.4;
+      } else {
+        spending += 2.9;
+      }
+      if (800 <= auto.mass && auto.mass <= 1100) {
+        spending += 1.7;
+      }
+      if (1101 <= auto.mass && auto.mass <= 1400) {
+        spending += 1.9;
+      }
+      if (auto.mass > 1400) {
+        spending += 2.2;
+      }
+    }
```

Comment:
Good work but a lot of code. In that cases - we try to create as many functions as possible to ommit repeating code

Consider this as an idea:

```
function fixCar() {
    // ...
    // we need to determine how the car is damaged/ruined 
    // and based on that we need to add corresponding values
}

function calculate() {
    // ...
    // using switch statements
}

function dealer(car) {
    const calculatePrice = fixCar(car.type) + defaultPrice
    return `Price for fixing your car ... ... ..${calculatePrice} UAH`
}
```